### ISSUE TITLE
Include post method for PlantPage to submit donations

### ISSUE THREAD

**Issue by Gab0rB** [Dica-Developer/weplantaforest#58]
For the first 3 buttons with the short list:

url: **/simpleDonateTrees**
the RequestBody should contain an object in the form of the example below:
```
{
  "targetAmountOfTrees": 10,
  "actualAmountOfTrees": 10,
  "actualPrice": 4797,
  "plantItems": [
    {
      "treeType": "Kiefer",
      "amount": 7,
      "treePrice": 570,
      "projectName": "Project 6 von admin"
    },
    {
      "treeType": "Buche",
      "amount": 1,
      "treePrice": 221,
      "projectName": "Project 5 von admin"
    }
  ]
}
```

For the custom button:

url: **/donateTrees**

The RequestBody should contain an object in the form of the example below:

```
{
  "targetPrice": 10000,
  "actualPrice": 9984,
  "projects": {
    "Project 6 von admin": {
      "plantItems": {
        "Buche": {
          "amount": 1,
          "treePrice": 223
        },
        "Birke": {
          "amount": 1,
          "treePrice": 511
        },
        "Kiefer": {
          "amount": 12,
          "treePrice": 570
        }
      }
    },
    "Project 7 von Johannes": {
      "plantItems": {
        "Buche": {
          "amount": 1,
          "treePrice": 360
        },
        "Birke": {
          "amount": 1,
          "treePrice": 179
        },
        "Kiefer": {
          "amount": 1,
          "treePrice": 429
        }
      }
    }
  }
}
```

**Comment by mschaaf** [comment]
When I post to simpleDonateTrees I get a 403 forbidden as response status. But there is no response body. Do you have an idea?

**Comment by mschaaf** [comment]
The first option request is returning forbidden so it seeme we did not have CORS enabled for posts.

**Comment by mschaaf** [comment]
This was the reason now it works. Great!

**Comment by mschaaf** [comment]
fixed with commit 9f75de51acb867fe0445ad4feb2d51fce6097598
